docs/internals/contributing/committing-code.txt

```diff
@@ -11,8 +11,8 @@ contribute code to Django, look at :doc:`writing-code/working-with-git` instead.
 Handling pull requests
 ======================
 
-Since Django is now hosted at GitHub, most patches are provided in the form of
-pull requests.
+Since Django is hosted at GitHub, patches are provided in the form of pull
```

Comment:
I'd say it's hosted "on" GitHub... ?

Reply:
Yeah, I stumbled over that, too, but I didn't want to make unrelated changes.

Reply:
As above I think it's fine to avoid retranslation